Documentation/MANPAGE.md

```diff
@@ -280,6 +280,13 @@ Run crypto speed test. Benchmark Go's built-in GCM against OpenSSL
 #### -trace string
 Write execution trace to file. View the trace using "go tool trace FILE".
 
+#### -use-random
+Use /dev/random for generating the master key instead of the default Go
+implementation. This is especially useful on embedded systems with Go versions
+prior to 1.9, which fall back to weak random data when the getrandom syscall
+is blocking. Using this option can block indefinitely when the kernel cannot
+harvest enough entropy.
+
```

Comment:
The "use-random" flag would be the only one that uses a hyphen "-" instead of an underscore "_". While I understand that this is what cryptsetup does, I think it is more important to be self-consistent here. I personally would just call it
`-devrandom` because it's clear, short, and works without a separator.